Scuti/Scripts/UI/MyAccountPresenter.cs

```diff
@@ -28,6 +28,13 @@ protected void OnSetData()
             locationText.text = Data.Location;
         }
 
+        public override void Open()
+        {
+            UIManager.SetFirstSelected(firstSelection);
+            base.Open();
+        }
```

Comment:
Why put this in the Open method if the base method in View already calls it? We can remove this line from pretty much all of these classes I think.  @Jpablogs8